src/App.js

```diff
@@ -1,24 +1,63 @@
-import logo from './logo.svg';
+import react, { useState } from 'react';
+
+import foodsJSON from "./foods.json";
 import './App.css';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm.jsx';
+import Searchbar from './components/Searchbar';
+
+
 
 function App() {
+  const foodArr = [...foodsJSON]
+  const [foodList, setFoodList] = useState(foodsJSON);
+  const [showFood, setShowFood] = useState(foodsJSON)
+  
+
+  const addFood = (food) => {
+    const foodListCopy = [...foodList, food];
+    setFoodList(foodListCopy)
+  };
+
+  const searchFood = (query) => {
+    const filteredFood = foodsJSON.filter((food)=> 
+    foodsJSON.name.toLowerCase().includes(query.toLowerCase())
+    );
+    setShowFood(filteredFood)
+  }
+
+  const deleteFood = (foodName) => {
+    const filteredFoodList = foodList.filter((food) => {
+      return food.name !== foodName;
+    })
+    setFoodList(filteredFoodList)
+    setShowFood(filteredFoodList)
+  }
+
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    {foodArr.map((food)=> {
+      return (
+        <div>
```

Comment:
Delete this `.map()` you're not supposed to render the list of food two times, in the lab instructions they ask you to replace this loop with the one that you have below that is already styled.